src/Astreiko.homework9.Nbrb.by.Tests/APIClientTests.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Astreiko.Homework9.Nbrb.@by.API_Client;
+using Astreiko.Homework9.Nbrb.@by.API_Client.Models;
+using Astreiko.Homework9.Nbrb.@by.FileClient;
+using NUnit.Framework;
+
+namespace Astreiko.homework9.Nbrb.by.Tests
+{
+    public class APIClientTests
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void GetRatesAsyncShouldGetRates()
+        {
+            //arrange
+            var apiClient = new APIClient();
+
+            //act
+            var rates = apiClient.GetRatesAsync(DateTime.Now, 141);
+
+            //assert
+            Assert.IsNotNull(rates);
+        }
+
+
+        [Test]
+        public void GetRatesAsyncShouldGetListRates()
+        {
+            //arrange
+            var apiClient = new APIClient();
+
+            //act
+            var listRates = apiClient.GetRatesAsync(new DateTime(2020,05,05), new DateTime(2020, 06, 06), 141);
+
+            //assert
+            Assert.IsNotNull(listRates);
+        }
+
+        [Test]
+        public void GetCurrenciesAsyncShouldGetListCurrency()
+        {
+            //arrange
+            var apiClient = new APIClient();
+
+            //act
+            var listCurrence = apiClient.GetCurrenciesAsync();
+
+            //assert
+            Assert.IsNotNull(listCurrence);
+        }
+
+        [Test]
+        public async Task SaveAsyncShouldCorrectlySerializeRate()
```

Comment:
Tests for different classes should be separated to be easily identifiable in tests structure (n test explorer, test reports, etc.)